### ISSUE TITLE
ScoutBase

### ISSUE THREAD

**Issue by omgimanerd** [stuyfission/stuyfission.github.io#68]

**Comment by jamesbchin** [comment]
This has to be done by competition.

**Comment by omgimanerd** [comment]
Not possible.

**Comment by noobbyte** [comment]
what exactly is this, a database of all teams?

**Comment by omgimanerd** [comment]
Yeah viewable by anyone, and people just submit their own data for scouting
purposes.

*Alvin Lin*
*Vice President of Stuyvesant FTC Fission 310*
*Head of Programming*
*http://omgimanerd.github.io <http://omgimanerd.github.io>*
